### DISCUSSION TITLE
The Budibase Discord server is now OPEN!!! 👾

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#7905] [Announcements]
You can join the Budibase Discord server using the link below:
https://discord.com/invite/ZepTmGbtfF

FYI - We will continue to support community members in Github Discussions as we always have!

**Comment by ankitaabad** [comment]
@joebudi This link is not working

**Reply by joebudi** [reply]
Hey.
The one above works for me. 

Maybe try this:
https://discord.gg/ZepTmGbtfF
